// console.log('ashish');
// response is in then() and error is the catch() and is better tutorial is in finally()
fetch('https://something.com').then().catch().finally()

// how to make promise is a object
// promise no. 1

const promiseOne = new promise()
